src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -413,7 +413,7 @@ private void deleteAllPublicationBlobData(Artefact artefact) {
         azureBlobService.deleteBlob(getUuidFromUrl(artefact.getPayload()));
 
         // Try to delete the generated files for the publications if it's not a flat file
-        if (!artefact.getIsFlatFile()) {
+        if (!artefact.getIsFlatFile().equals(Boolean.TRUE)) {
```

Comment:
I think the constant should go first in this

Reply:
fixed.